src/ontology/mondo.Makefile

```diff
@@ -963,6 +970,25 @@ update-%-mappings: $(TMPDIR)/new-exact-matches-%.owl
 		make NORM
 		mv NORM $(SRC)
 
+tmp/subclass-axioms.owl: $(SRC)
+	$(ROBOT) filter --input $(SRC) --axioms SubClassOf --preserve-structure false --trim false \
+		--drop-axiom-annotations "oboInOwl:source=~'(DOID|ICD10CM|NCIT|OMIM|OMIMPS|Orphanet):.*'" \
```

Comment:
Is this the full list of components, e.g. does ICD11Foundation need to be added here?

Reply:
Whatever the correct list is, the question is good - it makes me wonder if we will ever remember to update this command here when new sources are added to Mondo ingest.

Reply:
For now it is correct though (I checked the [source](https://raw.githubusercontent.com/monarch-initiative/mondo-ingest/main/src/ontology/reports/sync-subClassOf.confirmed.tsv), no icd11 in there at the moment), but we need to create another issue to ensure that we will update this list when new sources are added!

Reply:
Made an issue here: https://github.com/monarch-initiative/mondo-ingest/issues/615, assigning this back to you.

Reply:
Why is ICD11Foundation not included in the subclass sync and therefore would be added here?

Reply:
I suspect that the we do not have right now any ICD 11 mappings in mondo, and therefore nothing in the subclass sync? Otherwise I don’t know. But feel free to just add the icd11 foundation prefix here!